frontend/src/store/FamLoginUserState.ts

```diff
@@ -150,6 +162,14 @@ const cacheUserAccess = async () => {
     }
 };
 
+const delegatedCachedData = (application_id: number): FamRoleDto[] => {
+    const delegatedCachedData = state.value.famLoginUser?.accesses?.find(key => key.auth_key === DELEGATED_ADMIN_ROLE)?.grants.find((item) => {
```

Comment:
`state.value.famLoginUser?.accesses` is `getUserAccess()`

Reply:
Thanks, I overlooked that method...